ecscluster/variables.tf

```diff
@@ -39,6 +39,12 @@ variable "security_groups" {
   default     = []
 }
 
+variable "custom_cluster_policies" {
```

Comment:
Nitpick - we just call this `policies` in the ASG module.  Can we call it `policies` here so it's consistent?

Reply:
Happy to make that change.
I purposefully named it `custom_cluster_policies` to distinguish it from the ASG. Wouldn't one be overriding the other? or does the `policies` ARNs list apply to both?

Reply:
Input variable renamed.